MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,152 +2,161 @@ import UIKit
     private var quizeCounter = 0
     private var bestQuizResult = (score: 0, date: "")
-    private var currentDate: String {
-        let dateFormatter = DateFormatter()
-        dateFormatter.dateFormat = "dd.MM.yy HH:mm"
-        return dateFormatter.string(from: Date())
-    }
+    private var currentDate: String { Date().dateTimeString }
 
     @IBOutlet private weak var counterLabel: UILabel!
     @IBOutlet private weak var imageView: UIImageView!
     @IBOutlet private weak var textLabel: UILabel!
     @IBOutlet private weak var noButton: UIButton!
     @IBOutlet private weak var yesButton: UIButton!
 
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask { .portrait }
+
     // MARK: - Lifecycle
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        customizeImageView()
         if let firstQuestion = questions.first {
             let firstQuizeStep = convert(model: firstQuestion)
             show(quize: firstQuizeStep)
         }
     }
 
+    private func customizeImageView() {
+        imageView.layer.masksToBounds = true
+        imageView.layer.cornerRadius = 20
+    }
+
     @IBAction private func yesButtonClicked(_ sender: UIButton) {
-        let currentQuestion = questions[currentQuestionIndex]
-        if currentQuestion.correctAnswer {
-            showAnswerResult(isCoorect: true)
-            rightAnswerCounter += 1
-        } else {
-            showAnswerResult(isCoorect: false)
-        }
-        yesButton.isUserInteractionEnabled = false
-        noButton.isUserInteractionEnabled = false
-        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
-            self.showNextQuestionOrResult()
-        }
+        check(userAnswer: true)
+        disableUserInteractionForButtons()
+        showNextQuestionOrResultWithDelay(seconds: 1)
     }
     
     @IBAction private func noButtonClicked(_ sender: UIButton) {
+        check(userAnswer: false)
+        disableUserInteractionForButtons()
+        showNextQuestionOrResultWithDelay(seconds: 1)
+    }
+
+    private func check(userAnswer: Bool) {
         let currentQuestion = questions[currentQuestionIndex]
-        if !currentQuestion.correctAnswer {
-            showAnswerResult(isCoorect: true)
+        if userAnswer == currentQuestion.correctAnswer {
+            showAnswerResult(isCorrect: true)
             rightAnswerCounter += 1
         } else {
-            showAnswerResult(isCoorect: false)
+            showAnswerResult(isCorrect: false)
         }
+    }
+
+    private func disableUserInteractionForButtons() {
         yesButton.isUserInteractionEnabled = false
         noButton.isUserInteractionEnabled = false
-        DispatchQueue.main.asyncAfter(deadline: .now() + 1) {
+    }
+
+    private func enableUserInteractionForButtons() {
+        yesButton.isUserInteractionEnabled = true
+        noButton.isUserInteractionEnabled = true
+    }
+
+    private func showNextQuestionOrResultWithDelay (seconds: Double) {
```

Comment:
```suggestion
    private func showNextQuestionOrResultWithDelay(seconds: Double) {
```

Перед круглой скобкой с перечислением параметров функции пробел не ставится :)